modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/config/StorageTarget.scala

```diff
@@ -126,6 +130,12 @@ object StorageTarget {
       props.put("transportMode", "http")
       props
     }
+
+    override def initializers: List[Fragment] =
```

Comment:
This is a bit dirty, because it bypasses the abstractions we use elsewhere, of using the `Statement` sealed trait, and instead use the very powerful `Fragment`.

I can possibly justify it by saying `Statement`s are concerned with the ever-repeating period operations of the loader, for which we like to have unit tests that they are executed in the correct order.  Whereas these fragments are one-off statements executed only when the app first starts.

But please tell me if you can think I should avoid this short-cut.

Reply:
I don't mind it. Extending `Statement` would make a bunch of stub methods in the other modules.